script/make-pretty

```diff
@@ -87,6 +141,39 @@ do_clang_check()
         | xargs -n3 -P"$OT_BUILD_JOBS" script/clang-format-check
 }
 
+do_clang_tidy_fix()
+{
+    echo -e '====================='
+    echo -e '     check c/c++'
+    echo -e '====================='
+
+    (mkdir -p ./build/cmake-tidy \
+        && cd ./build/cmake-tidy \
+        && THREAD_VERSION=1.2 cmake "${OT_CLANG_TIDY_BUILD_OPTS[@]}" ../.. \
+        && run-clang-tidy.py -header-filter='.*' -checks="${OT_CLANG_TIDY_CHECKS}" -j"$OT_BUILD_JOBS" "${OT_CLANG_TIDY_FIX_DIRS[@]}" -fix)
+}
+
+do_clang_tidy_check()
+{
+    echo -e '====================='
+    echo -e '     check c/c++'
+    echo -e '====================='
+
+    (
+        mkdir -p ./build/cmake-tidy \
+            && cd ./build/cmake-tidy \
+            && THREAD_VERSION=1.2 cmake "${OT_CLANG_TIDY_BUILD_OPTS[@]}" ../.. \
+            && run-clang-tidy.py -header-filter='.*' -checks="${OT_CLANG_TIDY_CHECKS}" -j"$OT_BUILD_JOBS" "${OT_CLANG_TIDY_FIX_DIRS[@]}" \
```

Comment:
I fetch this PR locally, but failed to run tidy:
```bash
~/src/openthread$ ./script/make-pretty clang-tidy
=====================
     check c/c++
=====================
invocation ['clang-tidy', '-list-checks', '-p=/usr/local/google/home/simonlin/src/openthread/build/cmake-tidy', '-checks=-*,readability-make-member-function-const,', '-']
Traceback (most recent call last):
  File "/usr/bin/run-clang-tidy.py", line 251, in main
    subprocess.check_call(invocation)
  File "/usr/lib/python2.7/subprocess.py", line 185, in check_call
    retcode = call(*popenargs, **kwargs)
  File "/usr/lib/python2.7/subprocess.py", line 172, in call
    return Popen(*popenargs, **kwargs).wait()
  File "/usr/lib/python2.7/subprocess.py", line 394, in __init__
    errread, errwrite)
  File "/usr/lib/python2.7/subprocess.py", line 1047, in _execute_child
    raise child_exception
OSError: [Errno 2] No such file or directory
Unable to run clang-tidy.
```
I think it was searching for command `clang-tidy` but can't find it in $PATH. The correct command should be `clang-tidy-10` on my local workstation. Can you run it locally?

Reply:
I think we might need to run these commands:
```bash
sudo ln -sf /usr/bin/clang-tidy-10 /usr/bin/clang-tidy
sudo ln -sf /usr/bin/clang-apply-replacements-10 /usr/bin/clang-apply-replacements
```